changelog/unreleased/4570

```diff
@@ -0,0 +1,6 @@
+Feature: Maintain scroll position when user scrolls and navigates in file/folder list
```

Comment:
`Feature` is not a valid prefix for Calens. Check the [Calens README.md](https://github.com/owncloud/android/blob/fix/duplicated_autouploads/changelog/README.md) to see the valid prefixes, I'd say in this case the best one is `Enhancement`